spec/factories/products.rb

```diff
@@ -88,9 +99,17 @@
             evaluator.system.activations << FactoryGirl.create(:activation, system: evaluator.system, service: product.service)
           end
         else
-          fail 'product_enabled_on_system requires a system'
+          fail 'activated requires a system'
         end
       end
     end
+
+    transient do
```

Comment:
You have two `transient` and two `after:create` blocks on the root level now.
I think it would be less confusing to just join them together?